libraries/lib-mixer/WideSampleSequence.h

```diff
@@ -63,30 +63,34 @@ class MIXER_API WideSampleSequence
     @return false when `mayThrow` is false and not all samples could be
        retrieved
     */
-   bool GetFloats(size_t iChannel, size_t nBuffers, float *buffers[],
-      sampleCount start, size_t len,
-      fillFormat fill = fillZero, bool mayThrow = true,
-      sampleCount * pNumWithinClips = nullptr) const
+   bool GetFloats(
+      size_t iChannel, size_t nBuffers, float* buffers[], sampleCount start,
+      size_t len, bool backwards = false, fillFormat fill = fillZero,
+      bool mayThrow = true, sampleCount* pNumWithinClips = nullptr) const
    {
       // Cast the pointers to pass them to Get() which handles multiple
       // destination formats
-      return Get(iChannel, nBuffers, reinterpret_cast<samplePtr*>(buffers),
-         floatSample, start, len, fill, mayThrow, pNumWithinClips);
+      return Get(
+         iChannel, nBuffers, reinterpret_cast<samplePtr*>(buffers), floatSample,
+         start, len, backwards, fill, mayThrow, pNumWithinClips);
    }
 
    //! Retrieve samples of one of the channels from a sequence in a specified
    //! format
    /*!
     @copydetails SampleTrack::GetFloats()
     @param format sample format of the destination buffer
+    @param backward retrieves samples from `start` to `start + len` if false,
+       else from `start` to `start - len` in reverse order.
```

Comment:
I understand the "from" is inclusive and the "to" is exclusive, but maybe you want to say that explicitly.